extensions/sftp/tests/ListSFTPTests.cpp

```diff
@@ -236,9 +235,8 @@ TEST_CASE_METHOD(ListSFTPTestsFixture, "ListSFTP list one file writes attributes
   testController.runSession(plan, true);
 
   auto file = src_dir + "/vfs/nifi_test/tstFile.ext";
-  auto mtime = std::filesystem::last_write_time(file).time_since_epoch().count()/1000000000;
   std::string mtime_str;
-  REQUIRE(true == utils::timeutils::getDateTimeStr(mtime, mtime_str));
+  REQUIRE(true == utils::timeutils::getDateTimeStr(utils::file::FileUtils::to_time_t(std::filesystem::last_write_time(file)), mtime_str));
```

Comment:
this is the only place where we want to convert the file_type_time because we will check againts the logs
(I think we could also refactor the getDateTimeStr to accept file_type_time aswell)